pipeline/train/train.sh

```diff
@@ -102,6 +102,16 @@ if [[ -z ${USE_CPU+x} ]]; then
   extra_params+=('local')
 fi
 
+# Enable log & metrics publication only when the tracking script is available
+if ! command -v parse_tc_logs &> /dev/null
+then
+  echo "### Weight & Biases publication script is not available."
+  PARSER=cat
+else
+  echo "### Weight & Biases publication is available."
+  PARSER="parse_tc_logs --from-stream -v --wandb-project=${src}-${trg} --wandb-group ${model_type}.${training_type} --wandb-run-name $TASK_ID"
```

Comment:
It might be better to move this code to a transform where we can properly retrieve the data like Taskcluster task group id and the experiment ID for the CLI arguments. Ideally, this task should not depend on Taskcluster as we usually separate task implementation and the orchestrator-specific code. This job can potentially be launched locally or using a different orchestrator( snakemake etc.)

Reply:
@bhearsum was against transform usage (see comment [here](https://github.com/mozilla/firefox-translations-training/pull/501#discussion_r1565928735)).

I'll probably move the naming arguments into the python code, as it will be easier to condition these parameters for taskcluster usage

Reply:
moving to the publisher is a good idea!